deployments/docker/agent.yaml

```diff
@@ -0,0 +1,32 @@
+# *Required* The access token for the org that you wish to send metrics to.
+signalFxAccessToken: {"#from": "/etc/signalfx/token"}
```

Comment:
Change this to `signalFxAccessToken: $SIGNALFX_ACCESS_TOKEN` and it will pull it from an envvar, which will be much easier to configure in a docker container.

Reply:
Change the associated docs above too.

Reply:
Do ENVVARs require brackes? e.g. ${SIGNALFX_ACCESS_TOKEN}

Reply:
Done